.github/workflows/test.yml

```diff
@@ -23,12 +23,15 @@ on:
 jobs:
   test:
     runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        node_version: [14, 16]
```

Comment:
Do we need to run tests on multiple versions of node?

Reply:
maybe not, then we should also bump the version of `package.json#engines.node` I think.

Reply:
Sorry, I'm not sure I understand why this conversation was resolved.  I was imagining that you would remove the `matrix`, only test against 16, and update the `engines` line too?